src/module_wrap.cc

```diff
@@ -1529,7 +1529,9 @@ void ModuleWrap::SetImportModuleDynamicallyCallback(
 void ModuleWrap::HostInitializeImportMetaObjectCallback(
     Local<Context> context, Local<Module> module, Local<Object> meta) {
   Environment* env = Environment::GetCurrent(context);
-  CHECK_NOT_NULL(env);  // TODO(addaleax): Handle nullptr here.
+  if (env == nullptr) {
+    return;
+  }
```

Comment:
```suggestion
  if (env == nullptr)
    return;
```

Just a nit, feel free to ignore :-)